snapshot/pkg/controller/snapshotter/snapshotter.go

```diff
@@ -38,15 +38,18 @@ import (
 )
 
 const (
+	snapshotMetadataTimeStamp        = "SnapshotMetadata-Timestamp"
+	snapshotMetadataPVName           = "SnapshotMetadata-PVName"
+	snapshotDataNamePrefix           = "k8s-volume-snapshot"
 	defaultExponentialBackOffOnError = true
 
 	// volumeSnapshot* is configuration of exponential backoff for
-	// waiting for snapshot operation to complete. Starting with 10
-	// seconds, multiplying by 1.2 with each step and taking 15 steps at maximum.
-	// It will time out after 12.00 minutes.
-	volumeSnapshotInitialDelay = 10 * time.Second
-	volumeSnapshotFactor       = 1.2
-	volumeSnapshotSteps        = 15
+	// waiting for snapshot operation to complete. Starting with 2
+	// seconds, multiplying by 2 with each step and taking 15 steps at maximum.
```

Comment:
volumeSnapshotFactor = 1.5 and volumeSnapshotSteps = 20, so it should be:

multiplying by 1.5 with each step and taking 20 steps at maximum.

Reply:
fixed